examples/snippets/modules/ROOT/examples/akkastreams-grpc-java/src/main/java/sensordata/Logger.java

```diff
@@ -5,16 +5,18 @@
 import cloudflow.akkastream.AkkaStreamlet;
 import cloudflow.akkastream.AkkaStreamletLogic;
 import cloudflow.akkastream.javadsl.RunnableGraphStreamletLogic;
+import cloudflow.streamlets.CodecInlet;
 import cloudflow.streamlets.StreamletShape;
 import cloudflow.streamlets.proto.javadsl.ProtoInlet;
 
 import sensordata.grpc.SensorData;
 
 public class Logger extends AkkaStreamlet {
-    private final ProtoInlet<SensorData> inlet = new ProtoInlet<SensorData>(
+    private final ProtoInlet<SensorData> inlet = new ProtoInlet<>(
             "in",
             SensorData.class,
-            true
+            true,
+            CodecInlet::logAndSkip
```

Comment:
The code here was not compiling. I'm not sure this is the correct fix for it, though.

Reply:
There is a Java API for this, `ProtoInlet.create` (it automatically uses `logAndSkip`), which is probably better for the example.

Reply:
Resolved when rebasing with the main branch.